hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonTableInputFormat.java

```diff
@@ -561,7 +567,7 @@ protected Expression getFilterPredicates(Configuration configuration) {
     return resultFilterredBlocks;
   }
 
-  private org.apache.carbondata.hadoop.CarbonInputSplit convertToCarbonInputSplit(Blocklet blocklet)
+  private CarbonInputSplit convertToCarbonInputSplit(DetailedBlocklet blocklet)
       throws IOException {
     blocklet.updateLocations();
     org.apache.carbondata.hadoop.CarbonInputSplit split =
```

Comment:
Why not just use `CarbonInputSplit` instead of `org.apache.carbondata.hadoop.CarbonInputSplit`